### ISSUE TITLE
Mixup of test and validation in "Guess the correlation" example.

### ISSUE THREAD

**Issue by GitHubGeniusOverlord** [mlverse/torch#925]
Hi!
In the https://torch.mlverse.org/start/guess_the_correlation/ first example to learn torch, the text says
"After every epoch, the model is run on the validation data, in “test mode”", then however, the code goes like this:
`fitted <- net %>%
  setup(
    loss = function(y_hat, y_true) nnf_mse_loss(y_hat, y_true$unsqueeze(2)),
    optimizer = optim_adam
  ) %>%
  fit(train_dl, epochs = 10, valid_data = test_dl)`

so if I am not mistaken it shouldnt have "valid_data = test_dl", but "valid_data = valid_dl". 

Because this is the place where the validation dataloader is needed. 
Thanks!

**Comment by dfalbel** [comment]
Thank you for reporting @GitHubGeniusOverlord
